### DISCUSSION TITLE
Face-to-face meeting at OSFF London

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#20] [Meeting Agenda]
Hi @finos/technical-steering-committee - who else will be at the OSFF event in London next week? If there are enough of us, we should try and make some progress on the contribution process project #19

**Comment by rikoe** [comment]
✋

**Reply by ColinEberhardt** [reply]
that was quick! see you there. Hopefully a few others will be around.

**Comment by robmoffat** [comment]
I'll be there - covid permitting 🤞

**Comment by eminty69** [comment]
Its a bit far for me to come from Canada so I won't be attending

**Reply by ColinEberhardt** [reply]
That's a shame - hopefully see you at some other event.

**Comment by jonfreedman** [comment]
I'm afraid I'm missing this one, I am registered but turns out I'm actually going to be on holiday
